### ISSUE TITLE
Suge Knight?

### ISSUE THREAD

**Issue by crowman69** [Kapyyy/ShungiteTest#2]

**Comment by Kapyyy** [comment]
https://www.twitch.tv/drdisrespect/clip/YawningAbnegateKoalaWutFace?filter=clips&range=30d&sort=time

**Comment by thelacasa** [comment]
No, no Suge Knight, I think hes locked up in prison. He's talking shungite. Anyways, its a 2 billion year old like rock, stone that protects against frequencies and unwanted frequencies that may be traveling in the air. So thats his story. They bought a whole bunch of stuff, put them around the la planeta. Little pyramids. Stuff like that.
